Sources/Rendering/OpenGL/CellArrayBufferObject/Constants.js

```diff
@@ -0,0 +1,10 @@
+export const CoordShiftAndScaleMethods = {
+  Disable: 0,
+  Auto: 1,
+  AlwaysAuto: 2,
+  Manual: 3,
+};
+
```

Comment:
Can we just always have it on. Makes the code simpler. In C++ we have a path where the float point array can be directly uploaded to the VBO so auto has some advantage there as if there is no scaling we can take that fast path. But in vtkjs we always are copying the points into a new array.

Reply:
I just pushed changes to alwayshave it on.